tests/setup.py

```diff
@@ -13,8 +13,8 @@
     package_data={'': ['*.md']},
     include_package_data=True,
     install_requires=[
-        'ducktape@git+https://github.com/redpanda-data/ducktape.git@cad0aeaa32da8ad43a297e50d6cdb215770fd10c',
-        'prometheus-client==0.9.0', 'pyyaml==6.0', 'kafka-python==2.0.2',
+        'ducktape@git+https://github.com/ivotron/ducktape.git@48498ecec438002c3cdee6a734746447288023e4',
```

Comment:
curious if you tried upstream ducktape (plus our patches) or just 0.11.x?

Reply:
it's 0.11.x plus our patches. We do plan to analyze moving to upstream, given that it contains a "deflake" feauture, and also given we're on pt2.0